README.rst

```diff
@@ -77,6 +77,8 @@ The adventurous can install the most recent development version directly from ou
 Basic usage
 ===========
 
+For the detailed documentation, please refer to [https://happy-algorithms-league.github.io/hal-cgp/](https://happy-algorithms-league.github.io/hal-cgp/).
```

Comment:
```suggestion
For detailed documentation, please refer to [https://happy-algorithms-league.github.io/hal-cgp/](https://happy-algorithms-league.github.io/hal-cgp/). Here we only provide a preview.
```
or something like that to keep the reader reading ;)

Reply:
Thanks. Done.